slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/presentationsession/dto/CreateSessionRequestDto.java

```diff
@@ -0,0 +1,12 @@
+package com.binarystudio.academy.slidez.domain.presentationsession.dto;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class CreateSessionRequestDto {
+
+	private UUID presentationId;
+
+}
```

Comment:
We really need the class with just one field?

Reply:
Yes, because all future changes will be less complicated